changelogs/fragments/5020-fix-returnall-and-directory-avoidance-for-gopass.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - passwordstore - fix returnall and directory avoidance for gopass (https://github.com/ansible-collections/community.general/pull/5020).
```

Comment:
```suggestion
  - passwordstore lookup plugin - fix ``returnall`` and directory avoidance for gopass (https://github.com/ansible-collections/community.general/pull/5020).
```

Reply:
Should 4955-fix-path-detection-for-gopass.yaml be changed then too? That was merged with just the phrase `passwordstore` instead of `passwordstore lookup plugin`

Reply:
Yes, good point. I missed that... Will fix it :)

Reply:
Fixed in https://github.com/ansible-collections/community.general/commit/c64dd16f1cfeafb1593ec2bfd0b6d6c7cce7d4cb.